index.html

```diff
@@ -1,77 +1,311 @@
+                </button>
+              </div>
+            </section>
           </li>
         </ul>
-      </div>
-    </section>        
-  </div>
-</body>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h1>About <br /> Myself</h1>
+        </div>
+        <div class="about-text">
+          <p>
+            Hello I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+          </p>        
+        </div>
+        <h3>
+          <a href="#" class="connect2">LET'S CONNECT</a>
+        </h3>
+        <div class="social-icons2">
+          <ul>
+            <li>
+              <a href="#">
+                <img src="images/twitter.png" alt="Connect on Twitter">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/I.png" alt="Connect on Twitter">
+                <span>
+                  <img src="images/n.png" alt="">
+                </span>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/connect-m.png" alt="Connect on IM">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/github.png" alt="Connect on GitHub">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/thumbs-up.png" alt="Two fingers">
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <button type="button">
+            <span class="seeport">Get my resume</span> 
+          </button>
+        </div>
+        <div class="language">
+          <div class="language-head">
+            <span>Languages</span>
+            <span><i class="fas fa-chevron-down"></i></span>
+          </div>
+          <ul>
+            <li>
+              <div class="java">
+                <img src="images/javascript.png" alt="Javascript" />
+                <span>JavaScript</span>
+              </div>
+            </li>
+            <li>
+              <div class="java">
+              <img src="images/html.png" alt="HTML" />
+              <span>HTML</span>
+            </div>
+            </li>
+            <li>
+              <div class="java">
+              <img src="images/css.png" alt="CSS" />
+              <span>CSS</span>
+            </div>
+            </li>
+          </ul>
+        </div>
+        <div class="frame">
+          <div class="skill">
+            <span>Frameworks</span>
+            <i class="fas fa-chevron-right"></i>
+          </div>
+          <div class="skill">
+            <span>Skills</span>
+            <i class="fas fa-chevron-right"></i>
+          </div>
```

Comment:
- [ ] As I already mentioned, you should have made frame and skill elements on the same level as the language element, on their own. Such a semantic approach will help browsers, developers, and crawlers to distinguish between different types of data.
- [ ] For the horizontal line, you can use border-bottom styling for the elements instead of <hr>.

Reply:
Please check, I already implemented the border-bottom in the code

I will need more explanation on "....being on the same level" the `<skill>` class is a child of `<frame>` and frame is on same level with `<language>` or maybe I don't exactly understand the comment. Please be kind to explain it to my level 😁